src/App.vue

```diff
@@ -286,7 +288,11 @@ export default {
 				this.setPageTitle(nextConversationName)
 				// Update current token in the token store
 				this.$store.dispatch('updateToken', to.params.token)
+			} else if (to.name === 'notfound') {
+				this.setPageTitle('')
+				this.$store.dispatch('updateToken', '')
```

Comment:
as someone not knowing what effect this code will have, would be nice to add a comment :-)

Reply:
updating the token should automatically change the pagetitle?

Reply:
Somehow I think we should still keep the current URL, page title and token in place and only render a "not found" content. The switching will be done anyway by the user.

Reply:
Doesn't work reliably with guests. They would re initialise the conversation immediately and continue chatting without being annoyed for a second, until we can really block people.